### PR TITLE
Removed all hardcoded gufe keys from GufeTokenizable tests in test suite

### PR DESCRIPTION
**Pull request by dotsdl**
We do not consider `gufe` keys to be stable across versions, though they should be stable in-process and generally across machines running the same software versions. We only rely on them being stable in-process, however.

### CODE REVIEW

File: gufe/tests/test_tokenization.py [hunk 1]
```diff
@@ -140,9 +139,6 @@ def test_to_shallow_dict_roundtrip(self, instance):
         # include `np.nan`s
         #assert ser == reser
 
-    def test_key_stable(self, instance):
```
Comment (IAlibay) [inline gufe/tests/test_tokenization.py:143]: We should do a key stability test - i.e. if you generate the key twice do you get the same value. 

Shall we change this test to that?

**Review by IAlibay — CHANGES_REQUESTED** [review]
As you mention, keys should be stable in the same python call so we should probably have a small test that checks that.

File: gufe/tests/test_tokenization.py (hunk 1 above)
Comment (dotsdl) [reply gufe/tests/test_tokenization.py:143]: Yes! This is a good idea; I'll implement this next.

Comment (dotsdl) [reply gufe/tests/test_tokenization.py:143]: @IAlibay please have a look at 1dca49d and let me know if this satisfies!

**Review by IAlibay — APPROVED** [review]
lgtm!

**Comment by dotsdl** [comment]
Gracias @IAlibay! Merging!

**Pull request merged**